node/flags/flags.go

```diff
@@ -181,6 +181,21 @@ var (
 		Value:    "180",
 		EnvVar:   common.PrefixEnvVar(EnvVarPrefix, "EXPIRATION_POLL_INTERVAL"),
 	}
+	ReachabilityPollIntervalSecFlag = cli.StringFlag{
+		Name:     common.PrefixFlag(FlagPrefix, "reachability-poll-interval"),
+		Usage:    "How often (in second) to check if node is reachabile from DA backend",
+		Required: false,
+		Value:    "300",
```

Comment:
should we do it a bit more frequent, like once per min? It needs to clearly more frequent than batch interval to be more useful